pkg/assets/handlers.go

```diff
@@ -182,16 +182,12 @@ type WebConsoleConfig struct {
 	LogoutURI string
 }
 
-func GeneratedConfigHandler(config WebConsoleConfig, h http.Handler) http.Handler {
+func GeneratedConfigHandler(config WebConsoleConfig) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if strings.TrimPrefix(r.URL.Path, "/") == "config.js" {
-			w.Header().Add("Cache-Control", "no-cache, no-store")
-			w.Header().Add("Content-Type", "application/json")
-			if err := configTemplate.Execute(w, config); err != nil {
-				util.HandleError(fmt.Errorf("unable to render config template: %v", err))
-			}
-			return
+		w.Header().Add("Cache-Control", "no-cache, no-store")
+		w.Header().Add("Content-Type", "application/json")
+		if err := configTemplate.Execute(w, config); err != nil {
```

Comment:
I know you didn't change this, but we should render the template once into a byte array when GeneratedConfigHandler is called and just write the byte array to responses, rather then re-render it on every request